### PR TITLE
Transformation on a driver level

### PR DESCRIPTION
**Pull request by cheretka**
Transformation on a driver level 
 - Scale
 - Rotate
 - Flip

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Flip.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.drivers.transformation;
+
+public class Flip implements PointTransformation {
+    private boolean horizontally = false;
+    private boolean vertically = false;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Flip.java]: unused fields

File: src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Flip.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    private int h = 1;
+    private int v = 1;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Flip.java]: it looks like Scale could be reuse) to perform fliping easily.

File: src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Scale.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package edu.kis.powp.jobs2d.drivers.transformation;
+
+public class Scale implements PointTransformation {
+    private double deltaX;
+    private double deltaY;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Scale.java:5]: Is Flip necessary? Hor. and ver. flips can be instances of Scale (maybe factory would be useful)

File: src/main/java/edu/kis/powp/jobs2d/drivers/TransformationDriver.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.transformation.Point;
+import edu.kis.powp.jobs2d.drivers.transformation.PointTransformation;
+
+public class TransformationDriver implements Job2dDriver {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/TransformationDriver.java:7]: The name could indicate the design pattern that is used here.

**Review by kowallus — CHANGES_REQUESTED** [review]
Only minor issues. However, in order to merge you need to resolve conflicts with the io2-dev branch.

File: src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Scale.java (hunk 3 above)
Comment (cheretka) [reply src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Scale.java:5]: done

**Pull request merged**